indico/modules/events/operations.py

```diff
@@ -212,7 +213,8 @@ def _log_event_update(event, changes, visible_person_link_changes=False):
         'contact_emails': 'Contact emails',
         'contact_phones': 'Contact phone numbers',
         'label': {'title': 'Label', 'type': 'string', 'attr': 'title'},
-        'label_message': 'Label message'
+        'label_message': 'Label message',
+        'own_map_url': 'Map URL'
```

Comment:
`{'title': 'Map URL', 'type': 'string'}`

Like this you get "old -> new" in the log instead of a diff-like view. I think for a URL this makes more sense.